src/app/course/course.module.ts

```diff
@@ -13,10 +14,11 @@ import { CourseDurationInputComponent } from './add-page/course-durarion-input/c
 import { CourseAuthorsInputComponent } from './add-page/course-authors-input/course-authors-input.component';
 import { AddCourseContainerComponent } from './add-page/add-course-container/add-course-container.component';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
-
+import { MatChipsModule } from '@angular/material/chips';
 
 @NgModule({
-  declarations: [CourseListComponent, AddCourseContainerComponent, CourseItemComponent, ConfirmationModalComponent, CourseDateInputComponent, CourseDurationInputComponent, CourseAuthorsInputComponent],
+  // tslint:disable-next-line: max-line-length

```

Comment:
Unnecessary ignoring of linter rules is not a good practice. You can easily follow this rule by adding new lines, no need to turn it off